devbox.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packages": [
+    "yq-go@latest",
+    "kubebuilder@latest",
+    "jq@latest",
+    "go@1.22.5",
+    "gotests@latest",
+    "act@latest",
+    "kubectl@latest",
+    "docker@latest",
+    "kustomize@4.3",
+    "git@latest",
+    "gettext@latest",
+    "wget@latest",
+    "cosign@latest",
+    "kubernetes-helm@latest",
+    "govulncheck@latest",
+    "gotools@latest",
+    "go-licenses@latest",
+    "ginkgo@latest",
+    "operator-sdk@latest",
```

Comment:
I think we should pin the version of operator sdk

Reply:
Ok, `1.36.1` seems to be the one.

Same caveat, we need a process for revisiting pinned version, they cannot stay pinned for ever.